device-types/Palo Alto/PA-5450.yaml

```diff
@@ -0,0 +1,44 @@
+---
+manufacturer: Palo Alto
+model: PA-5450
+slug: palo-alto-pa-5450
+part_number: PA-5450
+u_height: 5.0
+is_full_depth: true
+airflow: front-to-rear
+console-ports:
+  - name: console port (Micro USB)
```

Comment:
I will request a change to the PAN-PA-5400-MPC-A module type.  It will have the console connections.  As a note, there is no configuration setting or interface name reference for the console interface on a PA-5450.  The best we can do is to list the label name printed on the card.